marshal.go

```diff
@@ -9,6 +9,7 @@ import (
 	"encoding/binary"
 	"errors"
 	"fmt"
+	"github.com/gocql/gocql/serialization/float"
```

Comment:
it should be below with other gocql/serialization imports

Reply:
Done.